hdr_schemata/utils/create_example.py

```diff
@@ -9,11 +10,33 @@ def get_subItems(item):
         else " | ".join(item["type"])
     )
 
-def create_example(path):
+def create_template(path):
     data = json.load(open(f"{path}.structure.json"))
     example = {item["name"]: get_subItems(item) for item in data}
-    print(json.dumps(example, indent=6))
+    # print(json.dumps(example, indent=6))
+    with open(f"{path}.template.json", "w") as f:
+        json.dump(example, f, indent=6)
+
+def make_example(item):
+    if item["name"] == "structuralMetadata":
+        return None
+    if item.get("subItems"):
+        if item["is_list"]:
+            return [{subItem["name"]: make_example(subItem) for subItem in item["subItems"]}]
```

Comment:
is this right?, your doing the same thing in the else?

Reply:
yes, it's very subtly different.  In the `if` the result is put into an array `[]`, in the `else` it is just the object returned.